Wikipedia/Code/EventLoggingFunnel.m

```diff
@@ -28,8 +29,7 @@ - (NSDictionary*)preprocessData:(NSDictionary*)eventData {
 }
 
 - (void)log:(NSDictionary*)eventData {
-    SessionSingleton* session = [SessionSingleton sharedInstance];
-    NSString* wiki            = [session.currentArticleSite.language stringByAppendingString:@"wiki"];
```

Comment:
Could this have the effect of causing article specific events appear to come from wrong lang wiki? Not sure if we're even doing anything like that...